core/arch/arm/plat-rpi3/platform_config.h

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright (c) 2016, Sequitur Labs Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * Redistributions in binary form must reproduce the above copyright notice,
+ * this list of conditions and the following disclaimer in the documentation
+ * and/or other materials provided with the distribution.
+ *
+ * Neither the name of ARM nor the names of its contributors may be used
+ * to endorse or promote products derived from this software without specific
+ * prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+# define RPI3_CONFIG
+
+# if 0
+# ifdef CFG_TEE_CORE_TA_TRACE
+# undef CFG_TEE_CORE_TA_TRACE
+# define CFG_TEE_CORE_TA_TRACE 1
+# endif
+# endif
+
+# define CFG_USE_UBOOT_HOOKS
+
+# ifdef CFG_TEE_CORE_TA_TRACE
+# undef CFG_TEE_CORE_TA_TRACE
+# endif
+
+# define CFG_DEBUG_VM_ALL 0
```

Comment:
conf.mk, yes. What is is about, I have no idea, we should ask Phil. It looks like a quick hack to enable some more debug prints. I don't think that we should fiddle with trace levels like that. So therefore I lean more towards just remove this for now and let Phil send a patch later on if he needs it. What do you think @jforissier ?